web/libs/datamanager/src/components/Common/Dropdown/DropdownComponent.jsx

```diff
@@ -25,7 +25,13 @@ export const Dropdown = React.forwardRef(({ animated = true, visible = false, ..
   const calculatePosition = React.useCallback(() => {
     const dropdownEl = dropdown.current;
     const parent = triggerRef?.current ?? dropdownEl.parentNode;
-    const { left, top } = alignElements(parent, dropdownEl, align ?? "bottom-left");
+    const { left, top } = alignElements(
+      parent,
+      dropdownEl,
+      align ?? "bottom-left",
+      /* padding = */ 0,
+      /* openUpwardForShortViewport = */ false,
```

Comment:
but the main concern here is that this is the common component, so it should not force the behaviour, it should receive it from the consumer. common component can be used in other cases where this option should be true, like on the bottom of the page or in a scrollable areas.